src/Generator.php

```diff
@@ -28,6 +30,33 @@ public function generateFromPath($path)
             }
         }
 
+        // json files
+        foreach ($dir as $fileinfo) {
```

Comment:
Looks good, just some minor notes;

could you merge this foreach into the previous one?